src/components/ComponentPages/ChangePassword/UI/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Form, Input, Button, Tabs, Row, Col } from "antd";
+import styles from "./ChangePassword.module.scss";
+import messages from "../../../../messages";
+
+export default function ChangePasswordUI({onFinish,onFinishFailed}) {
```

Comment:
It's a best practice to have the same Component and the file name. So Change the name of the folder "UI" to the component Name. i.e. ChangePasswordUI.

Reply:
Done